### PR TITLE
🏗 Update Design Review issue template

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

**Comment by samouri** [comment]
Implements https://github.com/ampproject/amphtml/issues/36888#issuecomment-983843325

File: build-system/common/update-session-issues/template/design-review.js [hunk 1]
```diff
@@ -6,12 +6,18 @@ const sessionDurationHours = 1;
...
 const timeRotationUtc = [
+  [/* wed */ 3, '16:30', 'Africa/Europe/western Asia'],
+  [/* wed */ 3, '21:00', 'Americas'],
+  [/* wed */ 3, '16:30', 'Africa/Europe/western Asia'],
+  [/* wed */ 3, '21:00', 'Americas'],
+  [/* wed */ 3, '16:30', 'Africa/Europe/western Asia'],
+  [/* wed */ 3, '21:00', 'Americas'],
   [/* wed */ 3, '16:30', 'Africa/Europe/western Asia'],
   [/* wed */ 3, '21:00', 'Americas'],
   [/* thu */ 4, '01:00', 'Asia/Oceania'],
```
Comment (rsimha) [inline build-system/common/update-session-issues/template/design-review.js:17]: Does this line need to be removed?

**Review by rsimha — APPROVED** [review]

File: build-system/common/update-session-issues/template/design-review.js (hunk 1 above)
Comment (samouri) [reply build-system/common/update-session-issues/template/design-review.js:17]: I don't think so. The [first step](https://github.com/ampproject/amphtml/issues/36888#issuecomment-983843325) that came out of design review was simply a reduced cadence.

Comment (alanorozco) [reply build-system/common/update-session-issues/template/design-review.js:17]: Like @samouri said, this changes cadence.

**Review by samouri — APPROVED** [review]
we have 3 approvals, merge at your leisure

**Pull request merged**